meerk40t/gui/laserrender.py

```diff
@@ -500,7 +509,7 @@ def make_thumbnail(
             width = int(round(width * scale))
             height = int(round(height * scale))
         if image_width != width or image_height != height:
-            pil_data = pil_data.copy().resize((width, height))
+            pil_data = pil_data.resize((width, height))

```

Comment:
Unclear why we remove .copy() here and leave the else copy below.

Reply:
The resize() does the copy. When it was in giant mode it crashed there. The copy() is superfluous, since resize performs a copy into the resized element.

Reply:
The copy below is because a copy is still needed for the pil_data. But, in the case when you're resizing making a full copy then a smaller copy can kill the memory. You rather make a small copy. Then below when nothing was resized you just make a regular copy.